examples/wsclient/main.go

```diff
@@ -36,9 +35,9 @@ func main() {
 	callAccountBalanceWS(w)
 
 	// streaming methods
-	callOrderbookWSStream()
-	callFilteredOrderbookWSStream()
-	callTradesWSStream()
+	callOrderbookWSStream(w)
+	//callFilteredOrderbookWSStream(w)
+	//callTradesWSStream(w)
 
```

Comment:
Did you mean to leave the above 2 lines commented?

Reply:
fixed